plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1161,10 +1270,45 @@ int amdgpu_plugin_dump_file(int fd, int id)
 }
 CR_PLUGIN_REGISTER_HOOK(CR_PLUGIN_HOOK__DUMP_EXT_FILE, amdgpu_plugin_dump_file)
 
+static int restore_process(int fd, CriuKfd *e)
+{
+	struct kfd_criu_process_bucket *process_bucket;
+	struct kfd_ioctl_criu_restorer_args args;
+	uint8_t *priv_data;
+	int ret = 0;
+
+	pr_debug("Restore process\n");
+
+	ret = init_restorer_args(&args, KFD_CRIU_OBJECT_TYPE_PROCESS, 0, 1,
+			sizeof(*process_bucket) + e->process_entry->private_data.len);
+
+	if (ret)
+		goto exit;
+
+	process_bucket = (struct kfd_criu_process_bucket*)args.objects;
+	/* First private data starts after all buckets */
+	priv_data = (uint8_t *)args.objects + sizeof(*process_bucket);
+
+	process_bucket->priv_data_offset = sizeof(*process_bucket);
```

Comment:
Is priv_data_offset the offset from the start of args.objects or from the start of priv_data? Everywhere else it seems to be from the start of priv_data. So here that should be 0.

Reply:
ACK - yes, nice catch

Reply:
ACK